### PR TITLE
Document recent API changes.

### PR DESCRIPTION
**Pull request by artemii235**
@gcharang All changes are released. Please feel free to publish them after merging this PR :slightly_smiling_face:

### CODE REVIEW

File: docs/basic-docs/atomicdex/atomicdex-api.md [hunk 1]
```diff
@@ -581,6 +581,144 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
...
+**best_orders coin action volume**
+
+The `best_orders` method returns the best price orders that can fill the volume for all existing pairs with selected coin.
+
+#### Arguments
+
+| Structure       | Type                       | Description                                                                   |
+| --------------- | -------------------------- | ----------------------------------------------------------------------------- |
+| coin            | string                     | the ticker of the coin to get best orders                                     |
+| action          | string                     | whether to `buy` or `sel` the selected coin                                   |
```
Comment (gcharang) [inline docs/basic-docs/atomicdex/atomicdex-api.md]: is `sel` as written here correct, or should it be `sell`? cc: @sergeyboyko0791

Comment (sergeyboyko0791) [reply docs/basic-docs/atomicdex/atomicdex-api.md]: I guess it's a typo

**Review by sergeyboyko0791 — CHANGES_REQUESTED** [review]
Thank you for adding the `unspendable_balance` field to the doc.
Could you please fix a few typos?

File: docs/basic-docs/atomicdex/atomicdex-api.md (hunk 1 above)
Comment (artemii235) [reply docs/basic-docs/atomicdex/atomicdex-api.md]: It's typo indeed, will fix shortly :slightly_smiling_face:

File: docs/basic-docs/atomicdex/atomicdex-api.md [hunk 2]
```diff
@@ -379,64 +379,367 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
...
+
+:::
+
+#### Arguments
+
+| Structure       | Type                       | Description                                                                   |
+| --------------- | -------------------------- | ----------------------------------------------------------------------------- |
+| coin            | string                     | the ticker of the coin to get best orders                                     |
+| action          | string                     | whether to `buy` or `sel` the selected coin                                   |
+| volume          | string                     | whether to `buy` or `sel` the selected coin                                   |
```
[the remark this answers is not in the document]
Comment (artemii235) [reply docs/basic-docs/atomicdex/atomicdex-api.md:455]: True :slightly_smiling_face:

File: docs/basic-docs/atomicdex/atomicdex-api.md [hunk 3]
```diff
@@ -379,64 +379,367 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
...
+where order entry has the following structure
+
+| Structure                | Type              | Description                                                                          |
+| ----------------------   | ----------------- | ------------------------------------------------------------------------------------ |
+| coin                     | string            | the ticker of the coin                                                               |
+| address                  | string            | the address offering the trade                                                       |
+| price                    | string (decimal)  | the price the user is willing to buy or sell per one unit of the coin from request   |
+| price_rat                | rational          | the price in num-rational crate format                                               |
+| price_fraction           | object (fraction) | the price represented as an object                                                   |
+| maxvolume                | string (decimal)  | the maximum amount of `base` the offer provider is willing to sell                   |
```
[the remark this answers is not in the document]
Comment (artemii235) [reply docs/basic-docs/atomicdex/atomicdex-api.md:472]: It's `maxvolume`, the name inherited from old good MM1 :slightly_smiling_face:

**Comment by artemii235** [comment]
@gcharang @sergeyboyko0791 I've pushed the fixes, could you review the PR again, please?

**Review by gcharang — APPROVED** [review]
LGTM!

**Review by sergeyboyko0791 — APPROVED** [review]
Looks good!

File: docs/basic-docs/atomicdex/atomicdex-api.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sergeyboyko0791) [reply docs/basic-docs/atomicdex/atomicdex-api.md:472]: I see, thanks

**Pull request merged**